code/datums/mind.dm

```diff
@@ -101,6 +101,10 @@
 	martial_art = default_martial_art
 	init_known_skills()
 	set_assigned_role(SSjob.GetJobType(/datum/job/unassigned)) // Unassigned by default.
+	//SKYRAT EDIT START - OPFOR SYSTEM
+	SSopposing_force.give_opfor_datum(src)
+	opposing_force.give_action_button()
```

Comment:
Why does this give the OPFOR button to everyone, again?

Reply:
I didn't see a better way to do it than give it to all minds